crates/apub/src/lib.rs

```diff
@@ -218,3 +225,13 @@ async fn insert_received_activity(ap_id: &Url, data: &Data<LemmyContext>) -> Lem
   ReceivedActivity::create(&mut data.pool(), &ap_id.clone().into()).await?;
   Ok(())
 }
+
+async fn is_mod_or_admin(
+  person: &ApubPerson,
+  community_id: CommunityId,
+  context: &LemmyContext,
+) -> LemmyResult<()> {
+  InstanceActions::check_ban(&mut context.pool(), person.id, person.instance_id).await?;
```

Comment:
Should probably remove this check, and leave it to other functions.

Reply:
Done, and removed this method as its only called in two places.